src/map/atcommand.c

```diff
@@ -1887,6 +1887,63 @@ ACMD(hair_color)
 	return true;
 }
 
+ACMD(setzone)
+{
+	char zone_name[MAP_ZONE_MAPFLAG_LENGTH];
+	memset(zone_name, '\0', sizeof(zone_name));
```

Comment:
minor thing, because zone_name is string, enough to set to 0 only first byte if you really want it to clear